essentials-framework/Essentials Devices Common/Essentials Devices Common/VideoCodec/CiscoCodec/CiscoSparkCodec.cs

```diff
@@ -1068,7 +1073,7 @@ void DeserializeResponse(string response)
                         CodecStatus.Status.RoomPreset = existingRoomPresets;
 
                         // Generecise the list
-                        NearEndPresets = RoomPresets.GetGenericPresets(CodecStatus.Status.RoomPreset);
+                        NearEndPresets = RoomPresets.GetGenericPresets(existingRoomPresets).Select(a =>(CodecRoomPreset)a).ToList();
```

Comment:
this might better be done as `existingRoomPresets.Select(a => a.ConvertCodecPreset());`, if I'm understanding how this is working correctly.

Reply:
I feel like I did this specifically because I wanted to cast to a list of a specific type, but I'll have to look.